govtool/backend/sql/list-dreps.sql

```diff
@@ -44,7 +44,7 @@ SELECT DISTINCT ON (dh.raw)
   encode(va.data_hash, 'hex'),
   dr_deposit.deposit,
   DRepDistr.amount,
-  (DRepActivity.epoch_no - GREATEST(MAX(COALESCE(block.epoch_no, block_first_register.epoch_no)), lve.epoch_no)) <= DRepActivity.drep_activity AS active,
+  (DRepActivity.epoch_no - GREATEST(COALESCE(block.epoch_no, block_first_register.epoch_no), lve.epoch_no, newestRegister.epoch_no)) <= DRepActivity.drep_activity AS active,
```

Comment:
is `newestRegister` the same as DRep Update?

Reply:
newestRegister as either update or registration (deposit either null or > 0)

Reply:
Perfect